docs/pages/upgrading/cloud-cluster-updates.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Cloud Cluster Updates
+description: Provides a high-level overview of Teleport cluster updates on Cloud.
+---
+
+Teleport Enterprise (Cloud) clusters composed of Auth and Proxy Services are automatically
+kept up to date with patches and minor releases following the release schedule
```

Comment:
```suggestion
On Teleport Enterprise (Cloud) clusters, the Auth Service and Proxy Service are automatically
kept up to date with patches and minor releases following the release schedule
```

Otherwise, it sounds like not all Teleport Enterprise (Cloud) clusters are composed of Auth/Proxy services, and only those that are get updated.